docs/topic_guide/installation.rst

```diff
@@ -1,8 +1,8 @@
 .. _sunpy-topic-guide-installing:
 
-************
-Installation
-************
+*********************
+Advanced Installation
```

Comment:
Is it worth linking out to the installation tutorial at the top of this page, in case someone accidentally finds this page instead?

Reply:
Oh yeah, that's a great idea. I will add that.